app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -560,6 +580,44 @@ object GameState {
     fun getMaxBet(): Int {
         return players.maxOf { it.bet }
     }
+
+    fun getSmallBlindPlayer(): Player? {
+        if (players.size < 2) {
+            return null
+        }
+        return players[players.size - 2]
+    }
+
+    fun getBigBlindPlayer(): Player? {
+        if (players.size < 2) {
+            return null
+        }
+        return players[players.size - 1]
+    }
+
+    fun nextCurrentPlayer() {
+        var index = players.indexOfFirst { player: Player -> player == currentPlayer }
```

Comment:
This is simpler and will also work as these objects are compared by reference (exactly as you wrote by hand)

```suggestion
        var index = players.indexOf(currentPlayer)
```

Reply:
Done.